test/e2e-secondary-network/infra/secondary_network_configuration.yml

```diff
@@ -0,0 +1,14 @@
+interface_type:
```

Comment:
To be consistent with other YAML file names, use "-" as delimiter between words: secondary-network-configuration.yml